api/test/test_helper.rb

```diff
@@ -4,5 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-require 'opentelemetry'
+require 'simplecov'
+SimpleCov.start
+
+require 'opentelemetry/sdk'
```

Comment:
Should be just `opentelemetry` - this is the API test helper, not the SDK.

Reply:
oh I see what happened....I should pay more attention to what *path* that file was in :|